docs-ref-autogen/azure-devops-extension-api/GitRestClient.yml

```diff
@@ -219,9 +219,8 @@ methods:
     package: azure-devops-extension-api
     summary: >-
       Request that another repository's refs be fetched into this one. It syncs
-      two existing forks. To create a fork, please see the \<a
-      href="https://docs.microsoft.com/en-us/rest/api/vsts/git/repositories/create?view=azure-devops-rest-5.1"\>
-      repositories endpoint\</a\>
+      two existing forks. To create a fork, please see the
+      [repositories endpoint](/rest/api/vsts/git/repositories/create?view=azure-devops-rest-5.1).
```

Comment:
This link works, but both the original link and the edited link redirect to version 5.0 even though version 5.1 is available.

Reply:
Updated to use the redirected link.